src/client/sandbox/node/window.js

```diff
@@ -304,8 +309,15 @@ export default class WindowSandbox extends SandboxBase {
 
         if (nativeMethods.registerServiceWorker) {
             window.navigator.serviceWorker.register = (...args) => {
-                if (typeof args[0] === 'string')
-                    args[0] = getProxyUrl(args[0], { resourceType: stringifyResourceType({ isScript: true }) });
+                const url       = args[0];
```

Comment:
Move this code to the `_isSecureOrigin (url)` and 
add link https://www.chromium.org/Home/chromium-security/prefer-secure-origins-for-powerful-new-features as comment inside function